docs/current_docs/integrations/kubernetes.mdx

```diff
@@ -4,73 +4,17 @@ slug: /integrations/kubernetes
 
 # Kubernetes
 
-## Deployment with Helm
+This section covers different strategies for deploying Dagger on a Kubernetes cluster.
+Its primary focus are teams that looking forward to self-host their CI infrastructure and
+use Dagger at its core to run their pipelines.
```

Comment:
Could we add more about why one would want to do this?

Reply:
Thanks Marcos. Much clearer why you'd wnat to use this now. However, could we move it to the top of the page? It feels a bit buried at the bottom right now.  I think we want someone coming to the docs for an integration to know immediately why they should bother setting this up and what the goals are.  

Could also do something like this:
Deploy Dagger Engine on your Kubernetes cluster to allow self-hosted CI runners run Dagger based pipelines. Enjoy the benefits of a shared global cache across all your Kubernetes hosted runners for faster builds.